modules/microshift-cni.adoc

```diff
@@ -10,12 +10,38 @@ OVN-Kubernetes is the default networking solution for {product-title} deployment
 
+.{product-title} network topology
+image:317_RHbM_OVN_topology_0323.png[title="{product-title} uses an overlay-based networking implementation, details follow."]
+
+[id="microshift-description-ovn-logical-components_{context}"]
+=== Description of the OVN logical components of the virtualized network
+* *OVN node switch:* A virtual switch named `<node-name>`.
+* *OVN cluster router:* A virtual router named `ovn_cluster_router`, also known as the distributed router.
+* *OVN join switch:* A virtual switch named `join`.
+* *OVN gateway router:* A virtual router named `GR_<node-name>`, also known as the external gateway router.
+* *OVN external switch:* A virtual switch named `ext_<node-name>.`
+* In this example, the `node-name` is `microshift-dev` and the cluster network is `10.42.0.0/16`.
+
+[id="microshift-description-connections-network-topology_{context}"]
+=== Description of the connections in the network topology figure
+* The north-south traffic between the network service device `enp1s0` and the OVN external switch `ext_microshift-dev`, is provided through the OVS patch port by the gateway bridge `br-ex`.
+* The OVN gateway router `GR_microshift-dev` is connected to the external network switch `ext_microshift-dev` through the logical router Port 4. Port 4 is attached with the node IP address 192.168.122.14.
+* The join switch `join` connects the OVN gateway router `GR_microshift-dev` to the OVN cluster router `ovn_cluster_router`. The IP address range is 100.62.0.0/16.
+** The OVN gateway router `GR_microshift-dev` is connected to the OVN join switch `join` through the logical router Port 3. Port 3 is attached with the internal IP address 100.64.0.2.
+** The OVN cluster router `ovn_cluster_router` is connected to the join switch `join` through the logical router port 2. Port 2 is attached with the internal IP address 100.64.0.1.
+* The OVN cluster router `ovn_cluster_router` is connected to the node switch `microshift-dev` through the logical router port 1. Port 1 is attached with the OVN cluster network IP address 10.42.0.1.
+* The east-west traffic between the pods and the network service is provided by the OVN cluster router `ovn_cluster_router` and the node switch `microshift-dev`. The IP address range is 10.42.0.0/24.
+* The east-west traffic between pods is provided by the node switch `microshift-dev` without network address translation (NAT).
+* The north-south traffic between the pods and the external network is provided by the OVN cluster router `ovn_cluster_router` and the host network. This router is connected through the `ovn-kubernetes` management port `ovn-k8s-mp0`, with the IP address 10.42.0.2.
+* All the pods, Pod 1 and Pod 2 in this example, are connected to the OVN node switch through their interfaces, `Interface name 1` and `Interface name 2` in this example.
```

Comment:
I'm also thinking this could be a sub bullet under line 44 since this sentence is referencing what being descried on line 44? Or maybe a starting bullet to refer to the example?